Timelines don't support comments after hideall

For example, `hideall "--sync--" # test` will cause an error in `npm run test`.

```
  1) timeline test
       ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.txt
         should load without errors:
     AssertionError: ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.txt:
4:Invalid format:hideall "--sync--" # test: expected { Object (lineNumber, line,
 ...) } to equal null
      at Context.<anonymous> (file:///C:/Users/enne/cactbot/test/helper/test_tim
eline.js:131:22)
      at processImmediate (internal/timers.js:461:21)
```

--- a/ui/raidboss/timeline_parser.ts
+++ b/ui/raidboss/timeline_parser.ts
@@ -139,7 +139,7 @@ export class TimelineParser {
       comment: /^\s*#/,
       commentLine: /#.*$/,
       durationCommand: /(?:[^#]*?\s)?(?<text>duration\s+(?<seconds>[0-9]+(?:\.[0-9]+)?))(\s.*)?$/,
-      ignore: /^hideall\s+\"(?<id>[^"]+)\"$/,
+      ignore: /^hideall\s+\"(?<id>[^"]+)\"(?:\s*#.*)?$/,
       jumpCommand: /(?:[^#]*?\s)?(?<text>jump\s+(?<seconds>[0-9]+(?:\.[0-9]+)?))(?:\s.*)?$/,
       line: /^(?<text>(?<time>[0-9]+(?:\.[0-9]+)?)\s+"(?<name>.*?)")(\s+(.*))?/,
       popupText:

